<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiFilterSettingWidget</class>
 <widget class="QWidget" name="MidiFilterSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="filterChannel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="channel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="filterType">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="type">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Note On</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Note Off</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Control Change</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Program Change</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pitch Bend</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aftertouch</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="filterCategory">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="category">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="filterProperty">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="property">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>channel</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>channel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterChannel</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterChannel</sender>
   <signal>toggled(bool)</signal>
   <receiver>channel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filterType</sender>
   <signal>toggled(bool)</signal>
   <receiver>type</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
